src/main/java/rx/internal/operators/TakeLastQueueProducer.java

```diff
@@ -36,9 +36,7 @@ public TakeLastQueueProducer(NotificationLite<T> n, Deque<Object> q, Subscriber<
         this.subscriber = subscriber;
     }
 
-    private volatile long requested = 0;
-    @SuppressWarnings("rawtypes")
-    private static final AtomicLongFieldUpdater<TakeLastQueueProducer> REQUESTED_UPDATER = AtomicLongFieldUpdater.newUpdater(TakeLastQueueProducer.class, "requested");
+    private AtomicLong requested = new AtomicLong();
```

Comment:
It's an internal class and a pattern already adopted in the project. The savings in the object overhead are usually worth it on a project so performance driven.